spec/routing/survey_routing_spec.rb

```diff
@@ -14,14 +14,13 @@
     expect(post('/surveys')).to route_to('surveys#create')
   end
 
-  # TODO(dinah): uncomment when surveys#update is implemented
-  # it "put surveys routes properly" do
-  #   expect(put("/surveys/1")).to route_to(
-  #     :controller => "surveys",
-  #     :action => "update",
-  #     :id => "1"
-  #   )
-  # end
+  it "put surveys routes properly" do
+    expect(put("/surveys/1")).to route_to(
+      :controller => "surveys",
```

Comment:
Also would be nice if we could change this to `:` from `=>`. Maybe you could change this whole file too.